src/App.vue

```diff
@@ -56,10 +56,19 @@ body {
 
 	--light-background: white;
 	--light-secondary-background: whitesmoke;
+	--light-selected-color: rgb(212, 212, 212);
+	--light-hover-color: whitesmoke;
```

Comment:
how about always using the same method to declare colors? how about settling to hex?